_archiv/2025/classes/edirom_introduction.html

```diff
@@ -2,7 +2,7 @@
 title: Introduction to Edirom-Online
 title-short: Introduction Edirom
 label: ediromE
-lang: english
+lang: deutsch/englisch
```

Comment:
Shouldn't that be "english"?
```suggestion
lang: deutsch/english
```

Reply:
thanks taking care about the review. i am totally fine with your suggestion. 
i just had a look to other courses in the past and adapted it https://ess.uni-paderborn.de/2024/programm.html